src/Security/Authentication/test/OpenIdConnect/OpenIdConnectTests.cs

```diff
@@ -294,6 +295,50 @@ public async Task SignOut_WithMissingConfig_Throws()
             Assert.Equal("Cannot redirect to the end session endpoint, the configuration may be missing or invalid.", exception.Message);
         }
 
+        [Fact]
+        public async Task SignOut_WithMissingIssuer()
```

Comment:
RemoteSignOut

Reply:
Add one positive test where the iss and sub are provided and you do get the Set-Cookie header.

Reply:
Did you mean iss and sid?